opensearch/src/test/java/org/opensearch/sql/opensearch/storage/OpenSearchStorageEngineTest.java

```diff
@@ -29,15 +33,33 @@ class OpenSearchStorageEngineTest {
   @Test
   public void getTable() {
     OpenSearchStorageEngine engine = new OpenSearchStorageEngine(client, settings);
-    Table table = engine.getTable("test");
+    Table table = engine.getTable(new CatalogSchemaName(".opensearch", "default"),"test");
     assertNotNull(table);
   }
 
   @Test
   public void getSystemTable() {
     OpenSearchStorageEngine engine = new OpenSearchStorageEngine(client, settings);
-    Table table = engine.getTable(TABLE_INFO);
+    Table table = engine.getTable(new CatalogSchemaName(".opensearch", "default"), TABLE_INFO);
     assertNotNull(table);
     assertTrue(table instanceof OpenSearchSystemIndex);
   }
+
+
+  @Test
+  public void getSystemTableForAllTablesInfo() {
+    OpenSearchStorageEngine engine = new OpenSearchStorageEngine(client, settings);
+    Table table
+        = engine.getTable(new CatalogSchemaName(".opensearch", "information_schema"), "tables");
```

Comment:
Do you plan to migrate existing implementation `source = .opensearch.information_schema.tables`?

Reply:
Not migration old `SHOW TABLES` command. we support existing commands and `source = @opensearch.information_schema.tables` will also be supported for list of tables.